docs/example.md

```diff
@@ -82,7 +100,40 @@ If the attributes and their values are identical then the connections are consid
 Connection attributes pass values from the connection dialog or the saved Tableau workbook to the _Connection Resolver_.
 In turn, the Connection Resolver uses these attributes to format an ODBC or JDBC connection string.
 
-![]({{ site.baseurl }}/assets/pce-tdr.png)
+```
+<tdr class='postgres_odbc'>
+    <connection-resolver>
+        <connection-builder>
+            // This is the component number 4 in the diagram above
+            <script file='connectionBuilder.js'/> 
+        </connection-builder>
+        <connection-properties>
+            // This is the component number 5 in the diagram above
```

Comment:
Can you add JDBC only in the comment here too

Reply:
Added